### PR TITLE
chore(external docs): Add `console` sink metadata

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

File: docs/reference/components/sinks/console.cue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  _features: {
+    batch: enabled: false
+    buffer: enabled: false
+    checkpoint: enabled: false
+    compression: enabled: false
+    encoding: {
+      enabled: true
+      default: null
+      ndjson: null
```
Comment (juchiast) [inline docs/reference/components/sinks/console.cue:16]: Currently, the code uses "json".

File: docs/reference/components/sinks/console.cue [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	}
+
+	features: {
+		batch: {
+			enabled:      false
+			common:       false
+			max_bytes:    null
+			max_events:   null
+			timeout_secs: 0
+		}
```
Comment (binarylogic) [inline docs/reference/components/sinks/console.cue:21]: This is my fault, but this should probably be:

```
batch: enabled: false
```

And that's it.

Comment (juchiast) [reply docs/reference/components/sinks/console.cue:21]: I think I had check-docs failing with missing values for common, max_bytes, etc.

Comment (juchiast) [reply docs/reference/components/sinks/console.cue:21]: `check-docs` fails with:
```
components.sinks.console.features.batch.common: incomplete value bool
components.sinks.console.features.batch.max_bytes: incomplete value >=0 & int | null
components.sinks.console.features.batch.max_events: incomplete value >=0 & int | null
components.sinks.console.features.batch.timeout_secs: incomplete value >=0 & <=255 & int
```

**Pull request merged**